lib/teiserver/coordinator/consul_commands.ex

```diff
@@ -754,7 +721,109 @@ defmodule Teiserver.Coordinator.ConsulCommands do
   def handle_command(%{command: "resetratinglevels", remaining: ""} = cmd, state) do
     ConsulServer.say_command(cmd, state)
     LobbyLib.cast_lobby(state.lobby_id, :refresh_name)
-    %{state | minimum_rating_to_play: 0, maximum_rating_to_play: 1000}
+    %{state | minimum_rating_to_play: 0, maximum_rating_to_play: LobbyRestrictions.rating_upper_bound()}
+  end
+
+  def handle_command(%{command: "resetchevlevels", remaining: ""} = cmd, state) do
+    ConsulServer.say_command(cmd, state)
+    LobbyLib.cast_lobby(state.lobby_id, :refresh_name)
+    %{state | minimum_rank_to_play: 0, maximum_rank_to_play: LobbyRestrictions.rank_upper_bound()}
+  end
+
+  #Reset min chev level for a lobby by using empty argument
+  def handle_command(%{command: "minchevlevel", remaining: ""} = cmd, state) do
+    ConsulServer.say_command(cmd, state)
+    LobbyLib.cast_lobby(state.lobby_id, :refresh_name)
+    %{state | minimum_rank_to_play: 0}
+  end
+
+  #Set min chev level for a lobby
+  def handle_command(
+        %{command: "minchevlevel", remaining: remaining, senderid: senderid} = cmd,
+        state
+      ) do
+    {allow, allow_msg} = LobbyRestrictions.allowed_to_set_restrictions(state)
+    if allow == :ok do
```

Comment:
I left a comment below regarding the janky signature `{:ok, nil} | {:error, reason}`. You can (and should) directly pattern match in the cond like so:
```elixir
case result do
  :ok -> ...
  {:error, reason} -> ...
end
```